webapp/src/components/Controls/DatasetSplitToggler.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { fireEvent, screen, within } from "@testing-library/react";
+import { renderWithRouterAndRedux } from "mocks/utils";
+import { DatasetSplitName } from "types/api";
+import DatasetSplitToggler from "./DatasetSplitToggler";
+import { setupServer } from "msw/node";
+import {
+  getDatasetInfo,
+  getDatasetInfoWithoutEvaluationSplit,
+  getDatasetInfoWithoutTrainSplit,
+} from "mocks/api/mockDatasetInfoAPI";
+
+const onChangeMock = (value: DatasetSplitName) => jest.fn();
+const renderDataSplitToggler = (value: DatasetSplitName) =>
+  renderWithRouterAndRedux(
+    <DatasetSplitToggler value={value} onChange={onChangeMock} />,
+    { route: "/local", path: "/:jobId" }
+  );
+
+describe("DatasetInfo API", () => {
+  const handlers = [getDatasetInfo];
+  const server = setupServer(...handlers);
+
+  beforeAll(() => server.listen());
+  afterEach(() => server.resetHandlers());
+  afterAll(() => server.close());
+
+  test("shoud have 2 toggle buttons displayed and enabled with icons", () => {
```

Comment:
Missing a `l`. Also, that's a details, but you are using the perfect sentence construction to use the alias `it()` instead of `test()`. By the way, I also really like that, as the output is super clear in case of failure. 👍 
```suggestion
  it("should have 2 toggle buttons displayed and enabled with icons", () => {
```
```
DatasetInfo API > should have 2 toggle buttons displayed and enabled with icons
```